services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/prometheus_metrics.py

```diff
@@ -0,0 +1,34 @@
+from typing import Annotated
+
+from fastapi import APIRouter, Depends, status
+from fastapi.responses import PlainTextResponse
+
+from ..modules.prometheus_metrics import UserServicesMetrics
+from ._dependencies import get_user_services_metrics
+
+router = APIRouter()
+
+
+@router.get(
+    "/metrics",
+    response_class=PlainTextResponse,
+    responses={
+        status.HTTP_500_INTERNAL_SERVER_ERROR: {
```

Comment:
I think 5xx are usually pretty clear they can happen. but I see these more as unexpected errors vs controlled errors. and 500 is pretty not defined... can be anything. are you sure you want this here?

Reply:
This is what prometheus expects, if it cannot retrieve a metric, a 500 with a short error description.